visualizations/plotly/webviz_plotly/__init__.py

```diff
@@ -19,6 +19,13 @@ class Plotly(JSONPageElement):
     Plotly page element. Arguments are the same as ``plotly.plot()`` from
     `plotly.js`. See https://plot.ly/javascript/ for usage.
 
+    :param xaxis: Will create a label for the x-axis.
```

Comment:
Is `xaxis` (and `yaxis`) good parameter names here for labels? I would suggest instead `xlabel` (or `xtitle`).